app/src/main/java/com/capstone/event_finder/network/EventViewModel.java

```diff
@@ -0,0 +1,36 @@
+package com.capstone.event_finder.network;
```

Comment:
💡 It's great that you introduced a concept of a `ViewModel`, but they are generally are used to hide the complexities of doing things like determining if you're using the API or the Database.

```
              ┌──────────────────────┐              
              │  Fragment / Activity │              
              └──────────────────────┘              
                          │                         
                          │                         
                          ▼                         
              ┌──────────────────────┐              
           ┌──│   Event View Model   │──┐           
           │  └──────────────────────┘  │           
           │                            │           
           ▼                            ▼           
┌────────────────────┐       ┌─────────────────────┐
│  Event Repository  │       │      Event API      │
└────────────────────┘       └─────────────────────┘
```

From the [Documentation](https://developer.android.com/reference/androidx/lifecycle/ViewModel): 
> ViewModel is a class that is responsible for preparing and managing the data for an [Activity](https://developer.android.com/reference/android/app/Activity.html) or a [Fragment](https://developer.android.com/reference/androidx/fragment/app/Fragment). **It also handles the communication of the Activity / Fragment with the rest of the application (e.g. calling the business logic classes).**

Additionally, your View Model interface should be simple, probably just have the two methods your fragments really care about:
* `getEvents()`
* `refreshEvents()`

The view model should then be the only one talking to the repository and API classes, and coordinating whether an API call is necessary, etc.

I don't think you need to fix it for this PR, but should look into following up for the MVP.

Reply:
Will look into this after the MVP! Thank you!